modules/mysql/main.tf

```
@@ -25,6 +25,12 @@ locals {
 
   databases = { for db in var.additional_databases : db.name => db }
   users     = { for u in var.additional_users : u.name => u }
+  iam_users = {
+    for user in var.iam_users : user.id => {
+      email         = user.email,
```

```suggestion
      username      = split("@", user.email)[0]
      email         = user.email
```
If the build still complains after this change, I plan on introducing a new property here to use for the [name](https://github.com/terraform-google-modules/terraform-google-sql-db/pull/467/files/792366a6e97b2a22630a2bae62e43f9878ecd0b4#diff-4993c81acf0e4cf2b6dbbc24454c673bbb4a74bb7db6f7fd1b728a09c590fc61R271) field:

```terraform
  name     = each.value.is_account_sa ? each.value.username : each.value.email
```